### PR TITLE
kafkacat: rename to kcat and update to kcat-1.7.0

### PR DESCRIPTION
**Pull request by TeddyDD**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/kcat/template [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+make_install_args="INSTALL=install"
+makedepends="librdkafka-devel yajl-devel"
 short_desc="Command line Apache Kafka producer and consumer"
 maintainer="Frank Steinborn <steinex@nognu.de>"
 license="BSD-2-Clause"
 homepage="https://github.com/edenhill/kafkacat"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=e67aecd153659d7ea0d561469a1e65038485fe9a3203fa8c9c59a9fe02164bf0
+checksum=baaed3cb6f5ade6a01355edb8819365e263a325375bee50727e009810e49cb38
+replaces="kafkacat>=1.6.0"
```
Comment (jcgruenhage) [inline srcpkgs/kcat/template:15]: The manual was missing a section on renaming packages, but assuming that @Chocimier opened https://github.com/void-linux/void-packages/pull/34875/files in response to this, I'd like to point you there. The way you renamed the package here is not sufficient.

**Review by jcgruenhage — CHANGES_REQUESTED** [review]
The manual was missing a section on renaming packages, but assuming that @Chocimier opened https://github.com/void-linux/void-packages/pull/34875/files in response to this, I'd like to point you there. The way you renamed the package here is not sufficient.

**Comment by Chocimier** [comment]
Yes, his needs to be improved. Thanks for bumping.

I haven't seen this, and written that section after it was suggested at #34581.

**Comment by jcgruenhage** [comment]
Ah, okay. I thought that you had written it in response to this because it came on the same day and specifically mentions the issues I've seen while going through this one here :D

**Comment by ahesford** [comment]
https://github.com/void-linux/void-packages/issues/36116

**Pull request closed without merging**